.github/workflows/build-bottles.yml

```diff
@@ -9,41 +9,72 @@ permissions:
   contents: write
 jobs:
   build-bottles:
-    runs-on: ${{ matrix.os }}
+    runs-on: ${{ matrix.os.id }}
 
     strategy:
+      fail-fast: false
       matrix:
-        include:
-          - os: macos-10.15
-            name: "Catalina"
-          - os: macos-11
-            name: "Big Sur"
+        os: [ { id: macos-10.15, name: "catalina" }, { id: macos-11, name: "big_sur" } ]
+        formula: [tezos-accuser-011-PtHangz2, tezos-admin-client, tezos-baker-011-PtHangz2, tezos-client, tezos-codec, tezos-endorser-011-PtHangz2, tezos-node, tezos-sandbox, tezos-signer]
```

Comment:
I slightly dislike the fact that list of these formulae is listed in two places :disappointed:

Annoyingly, YAML doesn't support any kind of "includes" of other files